templates/CRM/Member/Form/MembershipView.tpl

```diff
@@ -45,8 +45,14 @@
         {/if}
         {include file="CRM/common/formButtons.tpl" location="bottom"}
     </div>
+
+    {if $is_test}
+    <div class="help">
+      <strong>{ts}This transaction was submitted using a TEST payment processor{/ts}</strong>
```

Comment:
It is not always true that every test transaction is submitted using Test payment processor, it can be pay later as well.